appendices/common-arguments.md

```diff
@@ -0,0 +1,135 @@
+
+| Argument Name | Argument Value | Command Execution |
+| ------------- | -------------- | ----------------- |
+| `if` | true | yes |
+| `if` | false | no |
+
+### Example 1
+
+```G1ANT
+♥x = 3
+program notepad if ♥x==3
+```
+
+In the example above, number 3 is assigned to the `♥x` variable.  This part of the script: `program notepad if ♥x==3` will only be executed, if the condition is true. In this case, it is, therefore G1ANT.Robot **will open** Notepad.
+
+### Example 2
+
+```G1ANT
+♥x = theater
+program notepad if ♥x=="movie"
+```
+
+In the example above, the word “theater” is assigned to the `♥x` variable.  The second line of the script — `program notepad if ♥x=="movie"` — will only be executed, if the condition is true. In this case it is false (`theater` is not equal to `movie`), therefore G1ANT.Robot **will not** open Notepad.
+
+Please note that the word `movie` has to be in double quotes, because this is how strings have to be declared in C#.
+
+### Example 3
+
+```G1ANT
+♥x = 3
+♥y = theater
+program notepad if ⊂♥x==3 && ♥y=="theatre"⊃
+```
+
+G1ANT.Robot **will open** Notepad.
+
+## errorcall
+
+With the `errorcall` argument you can specify the procedure name to call in case an error occurs or a timeout expires.
+
+### Example
+
+```G1ANT
+window blah errorcall ErrorOccurred
+dialog OK!
+
+procedure ErrorOccurred
+    dialog ‴Could not find the specified window‴
+end procedure
+```
+
+In the first line of this example the robot tries to focus on the window titled *blah*. If it can’t find this window, the `ErrorOccurred` procedure is called, which shows an appropriate message.
+
+Note that when a user clicks OK in the dialog, the procedure ends, but the script continues: the robot goes back to where the procedure was called and displays another dialog box. This return to the place of a call in the script is the main difference between `errorcall` and `errorjump` arguments.
+
+## errorjump 
+
+The `errorjump` argument specifies name of a label to jump to when an error occurs during command’s execution or a timeout expires. It allows G1ANT.Robot to skip some part of the script and jump to the other.
+
+### Example 1
+
+In the example below some text is assigned to the `♥text` variable.
+
+The condition `⊂♥text == "Chris likes bananas"⊃` is false, so, because the `errorjump` argument was specified,  G1ANT.Robot jumps to `label banana` — the value for the `errorjump` argument. G1ANT.Robot will not show dialog with “*Yes! Chris likes bananas!*” message, because the `errorjump` argument is executed before any other command. Therefore, anything within the `banana` label will be executed. In this particular case a dialog message “*Sorry, Chris doesn't like bananas!*” is shown.
+
+```G1ANT
+♥text = ‴Chris likes apples‴
+test condition ⊂♥text == "Chris likes bananas"⊃ errorjump banana
+dialog ‴Yes! Chris likes bananas!‴
+label banana
+dialog ‴Sorry, Chris doesn't like bananas!‴
+```
```

Comment:
There's a minor issue in this code - if the condition would be true, then both dialogs would be displayed. it should be done as in the `Example 2`.